src/Microsoft.DotNet.PlatformAbstractions/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "description": "Abstractions for making code that uses file system and environment testable.",
-  "version": "1.1.10-servicing-*",
+  "version": "1.1.11-servicing-*",
```

Comment:
This never ships stable right? Is there need to bump the version?

Reply:
@eerhardt do we not need to bump this/DependencyModel?

Reply:
To make servicing (and understanding) easier, we've taken the approach that PlatformAbstractions and DependencyModel version with the rest of the branch.

For example: https://www.nuget.org/packages/Microsoft.Extensions.DependencyModel/

![image](https://user-images.githubusercontent.com/8291187/45760183-1e04a980-bbef-11e8-8dcf-34df0b99b0a3.png)

@safern is correct that we don't need to mark these packages as 'stable' when we aren't actually servicing them. However, I think it is still good practice to keep the version numbers all in sync in the branch. That way when looking at myget or other places of the packages produced by the branch, you can make sense of it all. So moving everything to `1.1.11-servicing-*` now is good. And then when we stablize `1.1.11`, we only mark stable the packages we want to ship. Then everything moves together to `1.1.12-servicing-*`.

Reply:
Sounds reasonable. I just wanted to understand why they were being bumped since when I was managing the servicing branches I recall I wouldn't bump those unless we were going to service them. Thanks for explaining 😄

Reply:
Yeah I'm OK with always versioning these as it does make it a little easier to reason about, especially in these servicing branches. However as already stated we shouldn't ship them unless they have changes.